### PR TITLE
make change

### PR DESCRIPTION
**Pull request by danmanners**

### CODE REVIEW

**Comment by danmanners** [comment]
plz review 🙏 @kashalls

**Review by bo0tzz — APPROVED** [review]

**Review by bo0tzz — APPROVED** [review]

**Review by bo0tzz — APPROVED** [review]

**Comment by bo0tzz** [comment]
:rocket:

**Review by kashalls — APPROVED** [review]
LGTM 👍

File: license.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+                    LOLWTFBBQ GENERAL PUBLIC LICENSE
+                       Version 1, 15 November 2023
+
+ No Copyright (C) 2023
+ Everyone is permitted to copy and distribute verbatim copies
+ of this license document, but changing it is not allowed.
```
Comment (bo0tzz) [inline license.md]:
```suggestion
 of this license document, but changing it is not allowed, with the exception of signing your name under the document.
```

File: license.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+                            Preamble
+
+Why in the hell would this be licensed?
+
+Seriously, _why are you here_?
+
+Go away, don't come back.
```
Comment (bo0tzz) [inline license.md:14]:
```suggestion
Go away, don't come back.

bo0tzz wuz here
```

**Review by bo0tzz — APPROVED** [review]

File: license.md (hunk 1 above)
Comment (danmanners) [reply license.md]: 1Like=1Pray

File: license.md (hunk 2 above)
Comment (danmanners) [reply license.md:14]: 2like=1pray, bad conversion rate 4 second commit

**Pull request merged**